contracts/interfaces/IDCAHubCompanion.sol

```diff
@@ -3,226 +3,8 @@ pragma solidity >=0.8.7 <0.9.0;
 
 import '@mean-finance/dca-v2-core/contracts/interfaces/IDCAHub.sol';
 import '@mean-finance/dca-v2-core/contracts/interfaces/IDCAPermissionManager.sol';
-import '@mean-finance/dca-v2-core/contracts/interfaces/IDCAHubSwapCallee.sol';
-import './IWrappedProtocolToken.sol';
 import './ISharedTypes.sol';
 
-interface IDCAHubCompanionParameters {
-  /// @notice Returns the DCA Hub's address
-  /// @dev This value cannot be modified
-  /// @return The DCA Hub contract
-  function hub() external view returns (IDCAHub);
-
-  /// @notice Returns the address of the wrapped token
-  /// @dev This value cannot be modified
-  /// @return The wToken contract
-  function wToken() external view returns (IWrappedProtocolToken);
-
-  /// @notice Returns the address used to represent the protocol token (f.e. ETH/MATIC)
-  /// @dev This value cannot be modified
-  /// @return The protocol token
-  // solhint-disable-next-line func-name-mixedcase
-  function PROTOCOL_TOKEN() external view returns (address);
-
-  /// @notice Returns the permission manager contract
-  /// @return The contract itself
-  function permissionManager() external view returns (IDCAPermissionManager);
-}
-
-interface IDCAHubCompanionSwapHandler is IDCAHubSwapCallee {
```

Comment:
We weren't using this. I think we forgot to remove it when we moved the swapper to a new contract